client/consensus/aura/src/lib.rs

```diff
@@ -143,7 +121,7 @@ impl SlotCompatible for AuraSlotCompatible {
 
 /// Start the aura worker. The returned future should be run in a futures executor.
 pub fn start_aura<B, C, SC, E, I, P, SO, CAW, Error>(
-	slot_duration: SlotDuration,
+	slot_duration: SlotDuration<u64>,
```

Comment:
I guess the idea was that it's being used in the public API and you don't have to import `sc_consensus_slots` additionally. Maybe we should re-export it?